### ISSUE TITLE
AMP: Missing aria-selected for role=option (Severity 5)

### ISSUE THREAD

**Issue by steinsebastian** [SAP/spartacus#14683]
Severity 5

AMP-Note: This INPUT (role=option) does not have an aria-selected attribute

Image attribute seems to have aria-selected attribute if it has role=option.
Also check for multi-selection-image-component: why is there no role=option?
![8-Missing aria-selected for role=option](https://user-images.githubusercontent.com/54309425/146955523-778e0489-067d-42e0-92e5-c0465d378bef.png)

**Comment by MichaelBoeser** [comment]
adding roles for multi-selection image made no difference in jaws, so I did not add any
